prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -395,6 +434,7 @@ has_detectors_reco MFT MCH && has_detector_matching MFTMCH && add_W o2-globalfwd
 
 if [ $BEAMTYPE != "cosmic" ]; then
   has_detectors_reco ITS && has_detector_matching PRIMVTX && add_W o2-primary-vertexing-workflow "$DISABLE_MC --disable-root-input $DISABLE_ROOT_OUTPUT $PVERTEX_CONFIG --pipeline $(get_N primary-vertexing MATCH REST)" "$PVERTEX_EXTRA_CONFIG"
+
```

Comment:
could you remove the unnecessary new line?

Reply:
Sure, will do.

Reply:
Done